modules/core/shared/src/main/scala/gem/math/EphemerisCoordinates.scala

```diff
@@ -4,12 +4,14 @@
 package gem.math
 
 import cats._
+import monocle.Lens
+import monocle.macros.GenLens
 
 /** A coordinate along with a rate of change in RA and Dec for some time unit,
   * expressed as an offset in p and q.  In reality the velocity information
   * comes from horizons and is always arcseconds per hour in horizons data.
   *
-  * @param coord coordinates
+  * @param coord delta
```

Comment:
Just curious why you changed this to be `delta` twice?

Reply:
Doh! I'll fix that. Thanks.